src/components/menu.js

```diff
@@ -1,18 +1,15 @@
 import React, { Component } from "react"
```

Comment:
Since it's no longer class based, would you mind removing the Component import? Otherwise everything looks great!